apps/ecommerce/frontend/src/helpers/fakeFetch.ts

```diff
@@ -0,0 +1,28 @@
+import { HydratedResourceData, ResourceLink } from '../types';
```

Comment:
Accidentally added this file to the commit. Whoops! But I will be adding this later. It's just abstracting out that fake data call